docs/experimental/extendable-param-files.md

```diff
@@ -1,27 +1,28 @@
-# Using the Extendable Bicep Parameters Feature (Experimental!)
+# Using the Extendable Bicep Parameters Feature
 
 ## What is it?
 
-Extendable Bicep Parameter Files is a feature that allows you to extend `.bicepparam` files from another `.bicepparam` file in order to reuse parameters across multiple deployments.
+Extendable Bicep Parameter Files (enabled with the `extend` keyword) is a feature that allows you to reuse parameters from one `.bicepparam` file in another `.bicepparam` file. This enables better re-use across multiple deployments and helps to keep your code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
 
 When using extendable bicep parameter files, you will have a main `.bicepparam` file that can be consumed by multiple extended `.bicepparam` files.
 
-## Using
+## Example Usage
 
 `main.bicep` This is your main bicep file, which will define your parameters for deployment.
 
 ```bicep
 param namePrefix string
 param location string
-param symbolicName string
+param tag string
 ```
 
 `root.bicepparam` This is your main bicepparam file, which can be reused by multiple extended .bicepparam files and in multiple deployments.
```

Comment:
Maybe I am the only one, but to me `root.bicepparam` file is the one being _extended_, not leaf.bicepparam. So when we say "can be reused by multiple extended .bicepparam files", I get confused.

Reply:
So the way I see it is that the root connects to multiple leaves - so the root can be reused multiple times into several different leaf files, which I think is also how Engin views it - should we ask the team?

Reply:
if I'm the only one with this perspective, it's fine to leave it as is